chapters/extend/executor.rst

```diff
@@ -105,14 +141,12 @@ Remember to add ``super().__init__(*args, **kwargs)`` to your :meth:`__init__`.
     All attributes declared in :meth:`__init__` will be persisted during :meth:`save`  and :meth:`load`.
 
 
-Override :meth:`post_init`
-^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 So what if the data you need to load is not in simple type. For example, a deep learning graph, a big pretrained model, a gRPC stub, a tensorflow session, a thread? The you can put them into :meth:`post_init`.
```

Comment:
This seems to be a question and I think you are missing 

```
So  what if the data you need to load is not in simple type.
```
I would write something like...
```
What if the data you need to load cannot be stored in a simple type?
```